tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
 flexmock
 ordereddict
 responses>=0.9.0
-pytest==3.2.5
-pytest-capturelog==0.7
-pytest-cov==2.5.1
+pytest==4.0.0
```

Comment:
do we really need exact version? How about `pytest==4.0.*` to keep compatibility but get bugfixes?

Reply:
That is a good idea :+1: 

Done.